ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/DummyPackageCommitter.scala

```diff
@@ -0,0 +1,23 @@
+package com.daml.ledger.participant.state.kvutils.committer
+
+import com.daml.ledger.participant.state.kvutils.committer.Committer.StepInfo
+import com.daml.metrics.Metrics
+
+/*
+ * This Package Committer do not run any validation but just prunes duplicated
+ *  packages from the entry.
+ *
+ * This should be used only by non distributed ledgers, like DAML-on-SQL,
+ * where the validation done in the API server is enough.
+ */
+private[committer] class DummyPackageCommitter(metrics: Metrics) extends PackageCommitter(metrics) {
```

Comment:
This name to me implies it doesn't really do anything, which isn't the case as it does build a log entry containing the packages. Perhaps a "NonValidatingPackageCommitter" would be more descriptive?

Reply:
changed.